design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -128,40 +130,63 @@ input GiftRegistryDynamicAttributeInput {
     value: String!
 }
 
-type CreateGiftRegistryOutput {
-    gift_registry: GiftRegistry
+interface GiftRegistryUserErrorsInterface {
+    status: Boolean!
```

Comment:
Needs to be the three-state enum we discussed elsewhere to allow for partial success.

Reply:
in this case no, because if one fails, no GR will be created.
there is no partial error.
I will remove status though.